### DISCUSSION TITLE
Pass Query Params to FileResponse

### DISCUSSION THREAD

**Discussion by sanhardik** [fastapi/fastapi#10038] [Questions]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
@app.get('/certs-filter')
def static_certs_filter(azure_hostname = None, server_description = None):
    if azure_hostname:
        return FileResponse(f'static/certs-filter.html?azure_hostname={azure_hostname}')
    elif server_description:
        return FileResponse(f'static/certs-filter.html?server_description={server_description}')
    else:
        return FileResponse('static/certs-filter.html')
```

### Description

I want to be able to pass some query params into the HTML returned as a FileResponse. 

I tried to send the query params on the path but that doesnt seem to work. 

What would be the right approach on this ?

### Operating System

Linux

### FastAPI Version

0.73.0

### Pydantic Version

1.9.0

### Python Version

Python 3.9.6

**Comment by YuriiMotov** [comment]
`path` parameter of `FileResponse` is path to the local file. It doesn't accept web URLs
